### PR TITLE
66-update brats bundle

### LINKED ISSUE #66: Brats bundle compatible issue
**Describe the bug**
Brats bundle is not compatible with 0.9.1.

**To Reproduce**
Infer with the following test command, you will get an error below.
`python -m monai.bundle run evaluating --meta_file configs/metadata.json --config_file configs/inference.json --logging_file configs/logging.conf`

**Error information**
```
Traceback (most recent call last):
  File "/opt/monai/monai/transforms/transform.py", line 90, in apply_transform
    return _apply_transform(transform, data, unpack_items)
  File "/opt/monai/monai/transforms/transform.py", line 54, in _apply_transform
    return transform(parameters)
  File "/opt/monai/monai/transforms/post/dictionary.py", line 642, in __call__
    transform_info = d[InvertibleTransform.trace_key(orig_key)]
KeyError: 'image_transforms'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/monai/monai/transforms/transform.py", line 90, in apply_transform
    return _apply_transform(transform, data, unpack_items)
  File "/opt/monai/monai/transforms/transform.py", line 54, in _apply_transform
    return transform(parameters)
  File "/opt/monai/monai/transforms/compose.py", line 173, in __call__
    input_ = apply_transform(_transform, input_, self.map_items, self.unpack_items, self.log_stats)
  File "/opt/monai/monai/transforms/transform.py", line 114, in apply_transform
    raise RuntimeError(f"applying transform {transform}") from e
RuntimeError: applying transform <monai.transforms.post.dictionary.Invertd object at 0x7faae6b6c130>

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/conda/lib/python3.8/site-packages/ignite/engine/engine.py", line 840, in _run_once_on_dataset
    self.state.output = self._process_function(self, self.state.batch)
  File "/opt/monai/monai/engines/evaluator.py", line 286, in _iteration
    engine.fire_event(IterationEvents.MODEL_COMPLETED)
  File "/opt/conda/lib/python3.8/site-packages/ignite/engine/engine.py", line 445, in fire_event
    return self._fire_event(event_name)
  File "/opt/conda/lib/python3.8/site-packages/ignite/engine/engine.py", line 421, in _fire_event
    func(*first, *(event_args + others), **kwargs)
  File "/opt/monai/monai/engines/workflow.py", line 234, in _run_postprocessing
    engine.state.batch[i], engine.state.output[i] = engine_apply_transform(b, o, posttrans)
  File "/opt/monai/monai/engines/utils.py", line 247, in engine_apply_transform
    transformed_data = apply_transform(transform, data)
  File "/opt/monai/monai/transforms/transform.py", line 114, in apply_transform
    raise RuntimeError(f"applying transform {transform}") from e
RuntimeError: applying transform <monai.transforms.compose.Compose object at 0x7faae6c89a90>
```

### PR DESCRIPTION
**Pull request by KumoLiu**
Signed-off-by: KumoLiu <yunl@nvidia.com>

Fixes #66  .

### Description
update brats bundle for 0.9.1

### Status
**Ready**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [x] Update `version` and `changelog` in `metadata.json`.
- [x] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.

### CODE REVIEW

**Review by wyli — APPROVED** [review]

File: models/brats_mri_segmentation/configs/metadata.json [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
+    "version": "0.1.1",
     "changelog": {
+        "0.1.1": "update for MetaTensor",
         "0.1.0": "complete the model package"
     },
-    "monai_version": "0.9.0",
-    "pytorch_version": "1.12.0",
-    "numpy_version": "1.21.2",
+    "monai_version": "0.9.1",
+    "pytorch_version": "1.13.0a0+340c412",
```
Comment (yiheng-wang-nv) [inline models/brats_mri_segmentation/configs/metadata.json]: here could it be a formal pytorch version? like 1.12.0, thus users (and our ci tests) are able to reproduce the environment.

Comment (KumoLiu) [reply models/brats_mri_segmentation/configs/metadata.json]: Thanks for your advice! I have just changed it.

**Review by Nic-Ma — APPROVED** [review]

**Comment by Nic-Ma** [comment]
Hi @yiheng-wang-nv @KumoLiu ,

Is there any blockers in the PR?

Thanks.

File: models/brats_mri_segmentation/configs/metadata.json (hunk 1 above)
Comment (yiheng-wang-nv) [reply models/brats_mri_segmentation/configs/metadata.json]: pytorch does not formally release 1.13.0, and we need to test the torchscript module withe the specified model. I will revert this merged PR, and may need your help to modify it again.

**Pull request merged**